go.mod

```diff
@@ -0,0 +1,11 @@
+module github.com/athenianco/cloud-common
+
+go 1.13
+
+require (
+	github.com/athenianco/metadata v0.0.0-20200619130750-022c72de9d66
```

Comment:
This kind of perverts the whole point of extracting a common library :smile:

Reply:
you guys are kidding